Sources/PackageGraph/ResolvedProduct.swift

```diff
@@ -66,11 +66,16 @@ public final class ResolvedProduct {
                                           dependencies: dependencies,
                                           packageAccess: true, // entry point target so treated as a part of the package
                                           testEntryPointPath: testEntryPointPath)
+            let (defaultLocalization, platforms) = if let firstTarget = targets.first {
+                (firstTarget.defaultLocalization, firstTarget.platforms)
```

Comment:
IIRC, we have code somewhere to merge the declared platforms for all targets (instead of just picking one), we should probably do the same here.

Reply:
Can you be a bit more specific? I didn't find such a thing when digging, but you know this code better than I do.

Reply:
I checked again and neither of these properties is target-specific, and there are comments elsewhere (e.g. [here](https://github.com/apple/swift-package-manager/blob/main/Sources/Build/BuildDescription/ProductBuildDescription.swift#L317) about them not being target-specific (and therefore "just use `[0]`"), so I'm a bit lost. Could use some direction. :)

Reply:
Oh! Found it. Two lines down.